### PR TITLE
squash docker-base image

### PR DESCRIPTION
**Pull request by lguohan**

### CODE REVIEW

File: slave.mk [hunk 1]
```diff
@@ -239,6 +239,15 @@ $(SONIC_INSTALL_WHEELS) : $(PYTHON_WHEELS_PATH)/%-install : .platform $$(addsuff
 docker-start :
 	@sudo service docker start &> /dev/null && sleep 1
 
+# targets for building squashed simple docker images that do not depend on any debian packages
+$(addprefix $(TARGET_PATH)/, $(SONIC_SQUASH_SIMPLE_DOCKER_IMAGES)) : $(TARGET_PATH)/%.gz : .platform docker-start $$(addsuffix -load,$$(addprefix $(TARGET_PATH)/,$$($$*.gz_LOAD_DOCKERS)))
```
Comment (marian-pritsak) [inline slave.mk]: Add them to DOCKER_LOAD_TARGETS and SONIC_CLEAN_TARGETS

Comment (lguohan) [inline slave.mk]: added

**Comment by lguohan** [comment]
address issue #124

**Comment by marian-pritsak** [comment]
```
Executing rules for target/docker-base.gz
docker: Error response from daemon: Conflict. The name "/docker-base.squash" is already in use by container 7076dc828321c1656cb1d7796ce8ad20af0dab42d2e58c7062813d28c092c714. You have to remove (or rename) that container to be able to reuse that name..
```

All Jenkins builds use the same docker engine, so containers are in same namespace.
We cannot run containers with same name, and we should also remove them after squash finishes.

**Comment by marian-pritsak** [comment]
New docker version 1.13 has --squash option, though its experimental
https://blog.docker.com/2017/01/whats-new-in-docker-1-13/

**Comment by lguohan** [comment]
so, we should probably upgrade to docker 1.13 in the sonic-slave

**Comment by lguohan** [comment]
addressed in #273

**Pull request closed without merging**